components/Admin/Admin.js

```diff
@@ -0,0 +1,65 @@
+import React, { PropTypes } from 'react'
+import AdminTopBar from '../AdminTopBar'
+import LocationList from '../LocationList'
+import history from '../../core/history'
+import { fetchLocations } from '../../core/firebaseApi'
+
+class Admin extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      locations: [],
+      filteredLocations: null,
+    }
+  }
+
+  // componentWillMount() {
+  //   fetchLocations()
+  //     .then(locations => {
+  //       this.setState({ locations })
+  //     })
+  // }
```

Comment:
@thebritican @adam-hart something is funky here.  If I leave this  function in, I get unknown errors.  Any thoughts?